packages/server/test/speckle-helpers/streamHelper.ts

```diff
@@ -113,10 +117,30 @@ export async function createTestStream(
   streamObj: BasicTestStream,
   owner: BasicTestUser
 ) {
-  const id = await createStream({
-    ...omit(streamObj, ['id', 'ownerId']),
-    ownerId: owner.id
-  })
+  let id: string
+  if (streamObj.workspaceId) {
+    const createWorkspaceProject = createWorkspaceProjectFactory({
```

Comment:
if u want a multiregion project, do `createTestStream` w/ workspaceId set AND run tests in multiregion mode